src/map/chrif.cpp

```diff
@@ -1330,6 +1330,7 @@ int chrif_save_scdata(struct map_session_data *sd) { //parses the sc_data of the
 				data.tick = 0; //Negative tick does not necessarily mean that sc has expired
 		} else
 			data.tick = INFINITE_TICK; //Infinite duration
```

Comment:
you shouldn't have `tick > tick_total`, only data.tick_total should be set as INFINITE_TICK